seminare/content/urls.py

```diff
@@ -3,6 +3,6 @@
 from seminare.content import views
 
 urlpatterns = [
-    path("pages/<slug>/", views.PageDetailView.as_view(), name="page_detail"),
+    path("pages/<path:slug>/", views.PageDetailView.as_view(), name="page_detail"),
```

Comment:
vedeli by sme tu mat len `<path:slug>/` a dat to ako fallback url, ak vsetko ostatne zlyha?